e2e/search-bar-optimized.spec.ts

```diff
@@ -92,4 +92,30 @@ test.describe('Search bar optimized', () => {
 
     await expect(searchInput).toHaveValue('');
   });
+
+  test('should not exist when the user navigates to a different page from the root', async ({
+    page
+  }) => {
+    await page.getByTestId('curriculum-map-button').nth(0).click();
+
+    // Any of the searchbar elements wont be visible for a certain amount of time, thus the timeout.
+    await page.waitForTimeout(500);
+
+    await expect(page.getByTestId('search-optimized')).not.toBeVisible();
```

Comment:
Rather than looking for the absence of something and waiting arbitrarily, how about waiting for the non-optimized search bar to appear? i.e. add `search-unoptimized` to the other search component and look for that.